gperftools/plan.sh

```diff
@@ -27,6 +39,10 @@ do_build() {
       -i "src/pprof"
 }
 
+do_install() {
```

Comment:
I think this can be removed.  `fix_interpreter "src/pprof" core/coreutils bin/env` is part of the `do_build` phase, so this `do_install` is redundant.